sqlstore/src/main/resources/db/migration/V8000__archive_library_type.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE LibraryType ADD COLUMN archived boolean DEFAULT FALSE;
+UPDATE LibraryType SET archived = TRUE WHERE platformType = 'Illumina' AND description NOT IN ('Paired End', 'Mate Pair', 'Single End');
```

Comment:
Could someone explain the functionality to me please?  It might help with some of our performance concerns...

Also, would this not be best enforced at the DAO level?